src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -257,6 +255,7 @@ internal interface ISyntaxFactsService : ILanguageService
         bool IsLockStatement(SyntaxNode node);
         bool IsUsingStatement(SyntaxNode node);
         bool IsStatement(SyntaxNode node);
+        bool IsExecutableStatement(SyntaxNode node);
```

Comment:
What's the difference between this and "IsStatement"?

Reply:
The only difference is in VB which considers things like the header of an if block or even a method signature and its "end statement" to be statements. For VB this returns `is ExecutableStatementSyntax` as opposed to `is StatementSyntax`.

Reply:
Are you suggesting I should remove IsStatement and change its usage to IsExecutableStatement? I can do that, but probably not in a separate PR because this PR depends on IsExecutableStatement working properly.

Reply:
➡️ Removed IsStatement in dc69b593fd789990075e35344142defaf91c4f0f